src/lib/atoms/media/CHOP-Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'semantic-ui-react';
+
+const CHOPLogo = ({ ...props }) =>
+<Image
+  {...props}
+  className="chopLogo"
+  width="120px"
+  src="https://s3.amazonaws.com/nexus.chop.edu/chop-logo.png" />;
```

Comment:
Before I had this on an s3 that I created.  I wonder if its better to add this image as an asset? Thoughts?

Reply:
Makes sense. Is `lib\atoms` a good place for an `assets` folder? Or it should be directly in `lib`?

Reply:
I guess `lib/atoms/assets` can work. Not sure if we use these images/logos more than once or twice. (mostly in other atoms)